Update securing api and configuration section (#2722)
wazuh/wazuh-documentation: source/user-manual/api/configuration.rst

| black,white    | black         | Sets the behavior of RBAC. For example, in black mode, policies not included in the list **can be** executed, while in white mode they **cannot** be executed. |
+----------------+---------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------+

max_login_attempts
^^^^^^^^^^^^^^^^^^^^^^
+----------------------+---------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------+
| Allowed values       | Default value | Description                                                                                                                                                    |
+======================+===============+================================================================================================================================================================+

> | Any positive integer.| 5             | Sets a maximum limit for login attempts during a specified ``block_time``.                                                                                     |

Reviewer: ```suggestion
| Any positive integer | 5             | Set a maximum number of login attempts during a specified ``block_time`` number of seconds.                                                                                     |
```

+----------------------+---------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------+

block_time
^^^^^^^^^^^^^^^^^^^^^^
+----------------------+---------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| Allowed values       | Default value | Description                                                                                                                                                                                                            |
+======================+===============+========================================================================================================================================================================================================================+

> | Any positive integer.| 300           | Established period of time (in seconds) for doing login requests. If the established number of requests (``max_login_attempts``) is exceeded within this time limit, the IP is blocked until the end of the block time.|

Reviewer: ```suggestion
| Any positive integer | 300           | Established period of time (in seconds) to attempt login requests. If the established number of requests (``max_login_attempts``) is exceeded within this time limit, the IP is blocked until the end of the block time period.|
```
